clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/ops/CloneKubernetesAtomicOperation.groovy

```diff
@@ -48,18 +49,25 @@ class CloneKubernetesAtomicOperation implements AtomicOperation<Void> {
    * curl -X POST -H "Content-Type: application/json" -d  '[ { "cloneServerGroup": { "source": { "serverGroupName": "k8s-test-v000" }, "credentials":  "my-k8s-account" } } ]' localhost:7002/kubernetes/ops
    * curl -X POST -H "Content-Type: application/json" -d  '[ { "cloneServerGroup": { "stack": "prod", "freeFormDetails": "mdservice", "targetSize": "4", "source": { "serverGroupName": "k8s-test-v000" }, "credentials":  "my-k8s-account" } } ]' localhost:7002/kubernetes/ops
   */
-  // TODO(stevenkim): should return DeploymentResult object
   @Override
-  Void operate(List priorOutputs) {
+  DeploymentResult operate(List priorOutputs) {
     CloneKubernetesAtomicOperationDescription newDescription = cloneAndOverrideDescription()
 
     task.updateStatus BASE_PHASE, "Initializing copy of server group for " +
       "${description.source.serverGroupName}..."
 
-    new DeployKubernetesAtomicOperation(newDescription).operate(priorOutputs)
+    DeployKubernetesAtomicOperation deployer = new DeployKubernetesAtomicOperation(newDescription)
```

Comment:
Instead of using `new`, declare the deployer as an `@Autowire`d field. Like this: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy#L45-L46
You'll also want to apply `@Component` to `DeployKubernetesAtomicOperation` like we do here: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy#L35-L36
Once you make those changes, you can drop the assignment of `deployer.kubernetesUtil`.